### DISCUSSION TITLE
Identification of objects in Plotter()

### DISCUSSION THREAD

**Discussion by jkissi** [marcomusy/vedo#535] [Q&A] [answered]
Hi! quick question:
As you can see on the image, I highlight my classifications with boxes:

![boxes_edit](https://user-images.githubusercontent.com/8246894/141820904-87597749-4067-45b5-8a94-e58b81467672.png)

For each box, I work out the end points of each line and draw the line using the `Line()` object. When I extract the meshes in the Plotter object using  `vis.getMeshes(at=0, includeNonPickables=True)` , I get a list of `Line()` objects and I can't tell them apart:

![boxes_terminal_edit](https://user-images.githubusercontent.com/8246894/141820934-8851fa85-e0e5-49fb-beb9-c6b4a1f65797.png)

I would like to change the properties of specific boxes, dependent on user interaction. Is there a way for me to give the Line() objects some kind of property, like a name or value, so I can act on specific groups of lines, say change the colour of a specific box based on a click or whatever?

Cheers!
j

**Comment by marcomusy** [comment, marked as answer]
Hi, yes all vedo objects have a name which you can set and retrieve:
`any_obj.name`

**Reply by jkissi** [reply]
setting it is straightforward? `any_obj.name = "string"` for example?

**Reply by marcomusy** [reply]
yes!

**Reply by jkissi** [reply]
awesome, cheers!!
